packages/relay/src/RegistrationManager.ts

```diff
@@ -245,17 +259,23 @@ export class RegistrationManager {
   }
 
   _isRegistrationCorrect (): boolean {
-    return isRegistrationValid(this.lastMinedRegisterTransaction, this.config, this.managerAddress)
+    return isRegistrationValid(this.lastMinedRegisterTransaction?.eventData, this.config, this.managerAddress)
   }
 
-  async _queryLatestRegistrationEvent (): Promise<EventData | undefined> {
+  async _queryLatestRegistrationEvent (): Promise<CachedEventData | undefined> {
```

Comment:
maybe a separate PR: if we stop using events , then can use registrar  here too.

Reply:
Created `OG-743`